sway/input/seat.c

```diff
@@ -727,11 +711,8 @@ void seat_set_focus_warp(struct sway_seat *seat,
 		if (config->mouse_warping && warp && new_output != last_output) {
```

Comment:
Need a `container != NULL` check here, otherwise focusing workspaces is crashing.